content/docs/tasks/traffic-management/egress-gateway.md

```diff
@@ -0,0 +1,316 @@
+          ports:
+          - number: 80
+            name: http-port
+            protocol: HTTP
+          - number: 443
+            name: https
+            protocol: HTTPS
+          resolution: DNS
+        ---
+        apiVersion: networking.istio.io/v1alpha3
+        kind: VirtualService
+        metadata:
+          name: direct-through-egress-gateway
+        spec:
+          hosts:
+          - edition.cnn.com
+          gateways:
+          - istio-egressgateway
+          - mesh
+          http:
+          - match:
+            - gateways:
+              - mesh
+              port: 80
+            route:
+            - destination:
+                host: istio-egressgateway.istio-system.svc.cluster.local
+                port:
+                  number: 80
+              weight: 100
+          - match:
+            - gateways:
+              - istio-egressgateway
+              port: 80
+            route:
+            - destination:
+                host: edition.cnn.com
+                port:
+                  number: 80
+              weight: 100
+        EOF
+    ```
+
+1.  Send an HTTP request to http://edition.cnn.com/politics.
+
+    ```command
+    $ kubectl exec -it $SOURCE_POD -c sleep -- curl -sL -o /dev/null -D - http://edition.cnn.com/politics
+    HTTP/1.1 301 Moved Permanently
+    ...
+    location: https://edition.cnn.com/politics
+    ...
+
+    HTTP/1.1 200 OK
+    Content-Type: text/html; charset=utf-8
+    ...
+    Content-Length: 151654
+    ...
+    ```
+
+    The output should be the same as in the [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress-tls-origination/) task, without TLS origination.
+
+1.  Check the log of the _istio-egressgateway_ pod and see a line corresponding to our request. If Istio is deployed to the `istio-system` namespace, the command to print the log is:
+
+    ```command
+    $ kubectl logs $(kubectl get pod -l istio=egressgateway -n istio-system -o jsonpath='{.items[0].metadata.name}') egressgateway -n istio-system | tail
+    ```
+
+    We should see a line related to our request, similar to the following:
+
+    ```plain
+    [2018-06-14T11:46:23.596Z] "GET /politics HTTP/1.1" 301 - 0 0 3 1 "172.30.146.87" "curl/7.35.0" "ab7be694-e367-94c5-83d1-086eca996dae" "edition.cnn.com" "151.101.193.67:80"
+    ```
+
+    Note that we redirected only the traffic from the port 80 to the egress gateway, the HTTPS traffic to the port 443 went directly to _edition.cnn.com_.
+
+### Let's clean up
+
+Let's remove the previous definitions before proceeding to the next step:
+
+```command
+$ istioctl delete gateway istio-egressgateway
+$ istioctl delete serviceentry cnn
+$ istioctl delete virtualservice direct-through-egress-gateway
+```
+
+## Perform TLS origination by the egress `Gateway`
+
+Let's perform TLS origination by the egress `Gateway`, similar to the [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress-tls-origination/) task.
```

Comment:
Let's perform TLS origination with the egress ...

Reply:
Here it means that the gateway performs the TLS origination and not the sidecar proxy. I think "by" should be used to stress this point.

Reply:
How about this?

Let's perform TLS origination with the egress `Gateway`, similar to the [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress-tls-origination/) task. Note that in this case the TLS origination will be done by the egress Gateway server, as opposed to by the sidecar in the previous task.